src/libraries/Microsoft.Extensions.Logging.Console/src/SimpleConsoleFormatter.cs

```diff
@@ -157,8 +158,12 @@ private static string GetLogLevelString(LogLevel logLevel)
 
         private ConsoleColors GetLogLevelConsoleColors(LogLevel logLevel)
         {
+            bool isAndroidOrAppleMobile = RuntimeInformation.IsOSPlatform(OSPlatform.Create("ANDROID"))
+                                          || RuntimeInformation.IsOSPlatform(OSPlatform.Create("TVOS"))
+                                          || RuntimeInformation.IsOSPlatform(OSPlatform.Create("IOS"))
+                                          || RuntimeInformation.IsOSPlatform(OSPlatform.Create("MACCATALYST"));
```

Comment:
This should be a static readonly field instead of calculating it every time this method get called.